pytradfri/types.py

```diff
@@ -0,0 +1,17 @@
+"""Typing information."""
+from typing import TypedDict
+
+TypeDeviceInfo = TypedDict(
```

Comment:
Will we need this type in more places than in the device module? Otherwise I suggest we keep the definition there to make it easier to read and follow the code.

Reply:
An idea I had was to create a `LightType´ and a ´AirPurifierType´, which would include the ´TypeDeviceInfo´. But that's for a future PR so I'll move it.